app/api/public/cron/sync-users/route.ts

```diff
@@ -0,0 +1,40 @@
+import { NextRequest, NextResponse } from "next/server";
+import { loadQstash } from "../../../../../lib/qstash";
+
+export const GET = async (req: NextRequest) => {
+  const authHeader = req.headers.get("authorization");
```

Comment:
```suggestion
  const authHeader = req.headers.get("x-cron-secret");
```

Reply:
I used "authorization" because I found it here: https://vercel.com/docs/cron-jobs/manage-cron-jobs#securing-cron-jobs
Maybe Vercel sends the cron secret under this name. I don't know if it can be changed (maybe in the config file)

Reply:
Then perfect, I didn't know about this link. Good stuff.